### DISCUSSION TITLE
How can I restrict changing decisions

### DISCUSSION THREAD

**Discussion by codebyollie** [snapshot-labs/snapshot#2283] [Q&A]
I created Snapshot proposal. I can re-vote. I choose some points from all possible and vote. After the transaction in Metamask passed I can change my decision. But I can't find how to restrict this. Vote once and that's all

**Comment by samwhosung** [comment]
There currently isn't a way to restrict this unfortunately.

**Comment by metrox-eth** [comment]
Already answered in Discord and Telegram please don't cross post.
